front-end/src/App.tsx

```diff
@@ -41,11 +43,14 @@ function App() {
       <LoggedInUserContext.Provider value={loggedInUser}>
       <BrowserRouter>
         <div>
-          <AppNavBar loggedInUser={loggedInUser} setLoggedInUser={setLoggedInUser} />
-          <Container fluid={true}>
-            <Switch>
+          <Container>
+            <Row style={{padding: '5%'}} className="justify-content-md-center">
```

Comment:
is there a better way to have padding?

Reply:
I would prefer using `rem` as units rather than percent. IIRC from experience, I get misshapen layouts sometimes when I use %'s, because percentages can look undesirable at certain window sizes.

If the padding looks fine when you scale the window around, this is fine too

Reply:
I'll change it to use `rem`